src/core/public/mocks.ts

```diff
@@ -28,6 +28,7 @@ import { deprecationsServiceMock } from './deprecations/deprecations_service.moc
 import { themeServiceMock } from './theme/theme_service.mock';
 
 export { chromeServiceMock } from './chrome/chrome_service.mock';
+export { capabilitiesServiceMock } from './application/capabilities/capabilities_service.mock';
```

Comment:
I don't think exporting this mock is necessary; you can already get it from the `applicationServiceMock`:
```js
const { capabilities } = applicationServiceMock.createStartContract();
```

Reply:
TIL! thanks